app/src/main/java/br/com/nespressohelper/daitan/nespressohelper/MainActivity.java

```diff
@@ -51,201 +58,121 @@
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(6);
-        tracosCapsula2Array.add(5);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(5);
-        tracosCapsula2Array.add(5);
-        tracosCapsula2Array.add(5);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(3);
-        tracosCapsula2Array.add(3);
-        tracosCapsula2Array.add(3);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(0);
-        tracosCapsula2Array.add(3);
-        tracosCapsula2Array.add(0);
+    @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
+    public void updateDB() throws IOException {
+
+
+        BackEndComm comm = new BackEndComm();
+        String url = "http://10.0.2.2:8080/coffee/";
+        comm.get(url, this);
+
+//        Coffee coffee = new Coffee();
+//
+//        coffee.setName("Espresso Intenso Decaffeinato");
+//        coffee.setCapsules(1);
+//        coffee.setBars1(2);
+//        coffee.setBars2(0);
+//        coffee.setDescription("Cuidadosamente descafeinado, preserva todo o aroma, intensidade e sabor do nosso Espresso Intenso.");
+//
+//        // Adding the image to the database
+//        drawable = getDrawable(R.drawable.espressointensodecaf);
+//        bitmap = ((BitmapDrawable) drawable).getBitmap();
+//        stream = new ByteArrayOutputStream();
+//        bitmap.compress(Bitmap.CompressFormat.PNG, 100, stream);
+//        bitmapData = stream.toByteArray();
+//        coffee.setImage(bitmapData);
+//
+//        coffees.add(coffee);
+
+    }
+
+    public ArrayList<Integer> generateDrawablesIds() {
+
+        ArrayList<Integer> drawables = new ArrayList<>();
+        drawables.add(R.drawable.espressointensodecaf);
+        drawables.add(R.drawable.ristretto);
+        drawables.add(R.drawable.ristrettoardenza);
+        drawables.add(R.drawable.espresso);
+        drawables.add(R.drawable.espressodecaf);
+        drawables.add(R.drawable.espressointenso);
+        drawables.add(R.drawable.espressobarista);
+        drawables.add(R.drawable.cafeaulait);
+        drawables.add(R.drawable.capuccino);
+        drawables.add(R.drawable.caramellattemacchiato);
+        drawables.add(R.drawable.cortado);
+        drawables.add(R.drawable.lattemacchiato);
+        drawables.add(R.drawable.soycappuccino);
+        drawables.add(R.drawable.vanillalattemacchiato);
+        drawables.add(R.drawable.nestealimao);
+        drawables.add(R.drawable.nesteapessego);
+        drawables.add(R.drawable.chococino);
+        drawables.add(R.drawable.chococinocaramel);
+        drawables.add(R.drawable.mocha);
+        drawables.add(R.drawable.nescau);
+        drawables.add(R.drawable.cafematinal);
+        drawables.add(R.drawable.lungo);
+        drawables.add(R.drawable.chaitealatte);
+        drawables.add(R.drawable.marrakesh);
```

Comment:
You could use a xml array resource for this. This would keep the main activity code cleaner

See: https://developer.android.com/guide/topics/resources/more-resources.html